app/assets/stylesheets/refinery/report.scss

```diff
@@ -0,0 +1,70 @@
+.Report {
+  background-color: white;
+
+  .report {
+    .v2-banner {
+      padding-bottom: 2rem;
+    }
+
+    p {
+      margin: 0;
+    }
+
+    &__content {
+      display: grid;
+      grid-auto-flow: dense;
+      grid-template-columns: 6rem 1fr 6rem;
+
+      &-date {
+        color: $v3-color_gray-dark;
+        display: block;
+        font-family: $font_family-secondary-light;
+        grid-column-start: 2;
+        margin-bottom: 1rem;
+        margin-top: 5rem;
+      }
+
+      &-title {
+        color: $v3-color_green-medium;
+        font-size: $font_size-xlarge;
+        grid-column-start: 2;
+        margin-bottom: 1rem;
+      }
+
+      &-body {
+        color: $v3-color_gray-dark;
+        font-size: $font_size-small;
+        grid-column-end: 4;
+        grid-column-start: 2;
+        margin: 0 6rem 3rem 0;
+        padding: 0;
+      }
+
+      &-image {
+        @include media(large) {
+          width: 37rem;
```

Comment:
Instead of using all these media queries do you want to try using the aspect ratio box approach I used at https://github.com/MAPC/perfect-fit/commit/b11737767577b68a549ddb494c8e47b090e7c153 ?